feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -155,100 +163,103 @@ func configInterfaceDUT(i *oc.Interface, me, peer *attrs.Attributes, subintfinde
 }
 
 // Configure Network instance on the DUT
-func configNetworkInstance(name string, peer *attrs.Attributes) *oc.NetworkInstance {
+func configNetworkInstance(name string, intf string, id string) *oc.NetworkInstance {
```

Comment:
no need to repeat the string type. Writing the parameters as "(name, intf, id string)" is sufficient

Reply:
Done. in new commit.